WordPress/Classes/Services/NotificationSyncMediator.swift

```diff
@@ -25,7 +25,7 @@ let NotificationSyncMediatorDidUpdateNotifications = "NotificationSyncMediatorDi
 class NotificationSyncMediator {
     /// Returns the Main Managed Context
     ///
-    fileprivate let contextManager: ContextManager
+    fileprivate let contextManager: CoreDataStack
```

Comment:
Should we change all of the contextManager properties to be "coreDataStack" to be consistent with what it is?

Reply:
We can do that as soon as we decide to keep only the Swift initialization. Wdyt?

I just want to prevent changing that, then (I hope not, but...) for some reason reverting this change, then having to rename all `coreDataStack` back to `contextManager`.

And since we're still using `ContextManager.sharedInstance` all over the code it would be good to change that too just to keep it consistent.